awx/main/tasks.py

```diff
@@ -1598,6 +1601,11 @@ def pre_run_hook(self, job):
             if job.is_isolated() is True:
                 pu_ig = pu_ig.controller
                 pu_en = settings.CLUSTER_HOST_ID
+
+            if job.instance_group.is_containerized is True:
```

Comment:
Comparison of is True here may let the programmer think that the corollary of `is False` can be used. However, in `is_containerized` current implementation `None` can be returned when the objective is to convey `False`. I'd say either cast `is_containerized` to bool (my preferred solution) or change this to drop the `is True`